From 79745739ed1d8c35b5f065a8fd2f010bf61681bc Mon Sep 17 00:00:00 2001
From: Jin Wei <wei.a.jin@intel.com>
Date: Thu, 18 Oct 2012 17:00:48 +0800
Subject: fix valgrind ioctl warning.

BZ: 62797
[CLEAN]

Initialize local variable to avoid syscall ioctl warning.

Category: aosp improvement
Domain: AOSP-General
Origin: internal
Upstream-Candidate: yes
Change-Id: I086a3b50247af23abb0fe0e9306949e2902d8f6b
Orig-Change-Id: I116d047dbd0c8dbe55b5fb1fe3ab3fea988cd907
Signed-off-by: Jin Wei <wei.a.jin@intel.com>
---
 libs/binder/ProcessState.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libs/binder/ProcessState.cpp b/libs/binder/ProcessState.cpp
index c1e49bc..ef2beda 100644
--- a/libs/binder/ProcessState.cpp
+++ b/libs/binder/ProcessState.cpp
@@ -345,7 +345,7 @@ static int open_driver()
     int fd = open("/dev/binder", O_RDWR);
     if (fd >= 0) {
         fcntl(fd, F_SETFD, FD_CLOEXEC);
-        int vers;
+        int vers = 0;
         status_t result = ioctl(fd, BINDER_VERSION, &vers);
         if (result == -1) {
             ALOGE("Binder ioctl to obtain version failed: %s", strerror(errno));
-- 
1.7.4.1

